activerecord/lib/active_record/schema_migration.rb

```diff
@@ -24,10 +24,9 @@ def table_exists?
         ActiveSupport::Deprecation.silence { connection.table_exists?(table_name) }
       end
 
-      def create_table(limit=nil)
+      def create_table
```

Comment:
This is changing the API of a public method. We will need to deprecate this argument before removing.